package.json

```diff
@@ -1,108 +1,43 @@
 {
-  "name": "fd-vue-webapp",
-  "version": "1.0.0",
-  "description": "A Vue.js web application for Freedomotic",
-  "author": "Freedomotic Team",
+  "name": "freedomotic",
+  "version": "0.0.1",
```

Comment:
why you change name and version here?

Reply:
The project was created using quasar-cli, those are parts of the files created alongside the starter template.

Reply:
=